spyder/widgets/about.py

```diff
@@ -105,70 +161,136 @@ def __init__(self, parent):
             the <a href="http://www.famfamfam.com/lab/icons/silk/">FamFamFam
             Silk icon set</a> 1.3 (&copy; 2006 Mark James; CC-BY 2.5), and
             the <a href="https://www.kde.org/">KDE Oxygen icons</a>
-            (&copy; 2007 KDE Artists; LGPL 3.0+).</small>
+            (&copy; 2007 KDE Artists; LGPL 3.0+).
+            </p>
+            <p>
+            Splash screen photo by
+            <a href="https://unsplash.com/@benchaccounting?utm_source=
+            unsplash&utm_medium=referral&utm_content=creditCopyText">Bench
+            Accounting</a> on <a href="https://unsplash.com/?utm_source=
+            unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash
+            </a>
             </p>
             <p>
             See the <a href="{github_url}/blob/master/NOTICE.txt">NOTICE</a>
             file for full legal information.
             </p>
             </div>
             """).format(
-                spyder_ver=versions['spyder'],
-                revision=revlink,
-                website_url=__website_url__,
                 github_url=__project_url__,
-                trouble_url=__trouble_url__,
-                forum_url=__forum_url__,
-                python_ver=versions['python'],
-                bitness=versions['bitness'],
-                qt_ver=versions['qt'],
-                qt_api=versions['qt_api'],
-                qt_api_ver=versions['qt_api_ver'],
-                os_name=versions['system'],
-                os_ver=versions['release'],
                 font_family=font_family,
                 font_size=font_size,
             )
         )
-        self.label.setWordWrap(True)
-        self.label.setAlignment(Qt.AlignTop)
-        self.label.setOpenExternalLinks(True)
-        self.label.setTextInteractionFlags(Qt.TextBrowserInteraction)
-        self.label.setFixedWidth(350)
+        self.label_overview.setWordWrap(True)
+        self.label_overview.setAlignment(Qt.AlignTop)
+        self.label_overview.setOpenExternalLinks(True)
+        self.label_overview.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_overview.setContentsMargins(15, 0, 25, 0)
+
+        self.label_community.setWordWrap(True)
+        self.label_community.setAlignment(Qt.AlignTop)
+        self.label_community.setOpenExternalLinks(True)
+        self.label_community.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_community.setContentsMargins(15, 0, 25, 0)
+
+        self.label_legal.setWordWrap(True)
+        self.label_legal.setAlignment(Qt.AlignTop)
+        self.label_legal.setOpenExternalLinks(True)
+        self.label_legal.setTextInteractionFlags(Qt.TextBrowserInteraction)
+        self.label_legal.setContentsMargins(15, 0, 25, 0)
```

Comment:
This can be simplified with a `for` loop since you're doing exactly the same for the three labels. Something like

```python
for label in [self.label_overview, self.label_community, self.label_legal]:
    label.setWordWrap(True)
    label.setAlignment(Qt.AlignTop)
    label.setOpenExternalLinks(True)
    label.setTextInteractionFlags(Qt.TextBrowserInteraction)
    label.setContentsMargins(15, 0, 25, 0)
```